zstacklib/zstacklib/network/ovs/utils.py

```
@@ -0,0 +1,312 @@
+
+    Returns:
+        Network interface name.
+
+    Raises:
+        OvsError: If interface not found.
+    """
+    try:
+        check_bdf_format(bdf)
+        net_path = f'/sys/bus/pci/devices/{bdf}/net'
+        if_name = os.listdir(net_path)[0].split('_')[0]
+        return if_name
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)